### ISSUE TITLE
non-validating peer fails with 'Not Implemented' when trying to sign HELLO msg

### ISSUE THREAD

**Issue by jeffgarratt** [hyperledger-archives/fabric#1214]
The call in core/Peer below fails with 'Not Implemented' error.

```
if viper.GetBool("security.enabled") {
    sig, err := p.secHelper.Sign(msg.Payload)
    if err != nil {
        return fmt.Errorf("Error signing Openchain Message: %s", err)
    }
    // Set the signature in the message
    msg.Signature = sig
}
return nil
```

**Comment by jeffgarratt** [comment]
This blocks #1182 .  Thanks so much Angelo!

**Comment by adecaro** [comment]
@jeffgarratt, I guess this issue can be closed, right? :)

**Comment by jeffgarratt** [comment]
Closed by #1220
